### PR TITLE
adding new topic 'Data and APIs'

### PR DESCRIPTION
**Pull request by ll911**
## Summary
https://github.com/bcgov/devhub-app-web/issues/700

## Notable Changes 
- adding new topic 'Data and APIs'
- including github repos for api-guidelines
- including github repos for api-specs
- including link to BC Data Catalogue
- including link to BC Government API Registry

### CODE REVIEW

File: app-web/registry/data-and-apis.json [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  "sourceProperties": {
+    "sources": [
+      {
+        "sourceType": "github",
+        "sourceProperties": {
+          "url": "https://github.com/bcgov/api-guidelines",
+          "owner": "bcgov",
+          "repo": "api-guidelines",
+          "files": [
+            "government-of-british-columbia-api-guidelines.md#api-performance-management",
```
Comment (patricksimonian) [inline app-web/registry/data-and-apis.json]: this doesn't work as you think it does. `files` should just have the one md. file.

File: app-web/registry/data-and-apis.json [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+          "title": "BC Government API Registry",
+          "description": "The Province of British Columbia API Registry",
+          "image": "https://developer.gov.bc.ca/static/BCID_H_rgb_rev-20eebe74aef7d92e02732a18b6aa6bbb.svg"
+        }
+      },      
+      {
+        "sourceType": "github",
+        "sourceProperties": {
+          "url": "https://github.com/bcgov/api-specs",
+          "author": "bcgov",
```
Comment (patricksimonian) [inline app-web/registry/data-and-apis.json]: should be `owner` not `author`

File: app-web/registry/featured-cards.json [hunk 3]
```diff
@@ -11,6 +11,18 @@
               "url": "https://signing-web-devhub-prod.pathfinder.gov.bc.ca"
             }
           },
+          {
```
Comment (patricksimonian) [inline app-web/registry/featured-cards.json]: @sheaphillips thoughts on having this as a featured card?

File: app-web/registry/data-and-apis.json (hunk 1 above)
Comment (ll911) [reply app-web/registry/data-and-apis.json]: ah, ok.

**Comment by patricksimonian** [comment]
#717

**Pull request closed without merging**